lib/ohai/plugins/aix/os.rb

```diff
@@ -25,6 +25,6 @@
 
   collect_data(:aix) do
     os collect_os
-    os_version kernel[:version]
+    os_version shell_out("oslevel -s").stdout.split($/)[0]
```

Comment:
Could you add or modify the the existing test to include a shell out result with trailing whitespace?